lib/memoryjs.cc

```diff
@@ -146,6 +148,60 @@ Napi::Value findModule(const Napi::CallbackInfo& args) {
   }
 }
 
+Napi::Value findPattern(const Napi::CallbackInfo& args) {
+  Napi::Env env = args.Env();
+
+  // Address of findPattern result
+  uintptr_t address = -1;
+
+  // Define error message that may be set by the function that gets the modules
+  const char* errorMessage = "";
+
+  pid_t hProcess = (pid_t)args[0].As<Napi::Number>().Int64Value();
+
+  std::vector<module::Module> moduleEntries = module::getModules(hProcess, &errorMessage);
+
+  // If en error message was returned from the function getting the modules, throw the error.
+  // Only throw an error if there is no callback (if there's a callback, the error is passed there).
+  if (strcmp(errorMessage, "") && args.Length() != 7) {
+    Napi::Error::New(env, errorMessage).ThrowAsJavaScriptException();
+    return env.Null();
+  }
+
+  for (std::vector<module::Module>::size_type i = 0; i != moduleEntries.size(); i++) {
+    std::string moduleName(args[1].As<Napi::String>().Utf8Value());
+
+    char *match = strstr(moduleEntries[i].pathname, moduleName.c_str());
+
+    if (NULL == match) {
```

Comment:
Shouldn't this be !=, as in moduleName is a substring of path?

Reply:
Was trying to mimic [this line](https://github.com/Rob--/memoryjs/blob/79618efdefc2aac203ae7137eefa2cf30cf888ce/lib/memoryjs.cc#L677) from the original `memoryjs` repo.

```
    if (!strcmp(moduleEntries[i].szModule, moduleName.c_str())) {
```

Then again I'm not entirely why they did it like that.